.merlin

```diff
@@ -20,6 +20,7 @@ B _build/lib/bap_plugins
 B _build/lib/text_tags
 B _build/lib/regular
 B _build/lib/graphlib
+B _build/lib/microx
```

Comment:
The root `.merlin` file should only contain core libraries, like bap itself and its ancestors. 

This is the correct contents of the `B` section of `.merlin` in mine repository (not yet merged):

```
B _build
B _build/lib/bap
B _build/lib/regular
B _build/lib/graphlib
B _build/lib/bap_future
B _build/lib/bap_plugins
B _build/lib/bap_bundle
B _build/lib/bap_config
```

A code that uses some library, should create its own local `.merlin` that refers to it. If we will put everything into the root file, it will work very slowly.